components/infobox/wikis/brawlstars/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,233 @@
+	caster = {category = 'Casters', variable = 'Caster', isplayer = false, personType = 'talent'},
+	manager = {category = 'Manager', variable = 'Manager', isplayer = false, personType = 'staff'},
+	['director of esport'] = {
+		category = 'Director of Esport', variable = 'Director of Esport', isplayer = false, personType = 'staff'},
+	streamer = {category = 'Streamers', variable = 'Streamer', isplayer = false},
+	producer = {category = 'Production Staff', variable = 'Producer', isplayer = false, personType = 'talent'},
+	director = {category = 'Production Staff', variable = 'Director', isplayer = false, personType = 'talent'},
+	['content creator'] = {
+		category = 'Content Creators', variable = 'Content Creator', isplayer = false, personType = 'talent'},
+	captain = {category = 'Captain', variable = 'Captain', isplayer = false},
+}
+_ROLES['assistant coach'] = _ROLES.coach
+_ROLES['in-game leader'] = _ROLES.igl
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+function CustomPlayer.run(frame)
+	local player = Player(frame)
+
+	player.adjustLPDB = CustomPlayer.adjustLPDB
+	player.createBottomContent = CustomPlayer.createBottomContent
+	player.createWidgetInjector = CustomPlayer.createWidgetInjector
+	player.defineCustomPageVariables = CustomPlayer.defineCustomPageVariables
+	player.getStatusToStore = CustomPlayer.getStatusToStore
+	player.getPersonType = CustomPlayer.getPersonType
+
+	_args = player.args
+
+	return player:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'status' then
+		return {
+			Cell{name = 'Status', content = {CustomPlayer._getStatus()}},
+			Cell{name = 'Years Active (Player)', content = {_args.years_active}},
+			Cell{name = 'Years Active (Org)', content = {_args.years_active_manage}},
+			Cell{name = 'Years Active (Coach)', content = {_args.years_active_coach}},
+		}
+	elseif id == 'role' then
+		return {
+			Cell{name = 'Role', content = {
+				CustomPlayer._createRole(_args.role),
+				CustomPlayer._createRole(_args.role2)
+			}},
+		}
+	elseif id == 'history' then
+		table.insert(widgets, Cell{
+			name = 'Retired',
+			content = {_args.retired}
+		})
+	end
+	return widgets
+end
+
+function CustomInjector:addCustomCells(widgets)
+	if String.isEmpty(_args.mmr) then
+		return {}
+	end
+
+	local mmrDisplay = '[[Leaderboards|' .. _args.mmr .. ']]'
+	if String.isNotEmpty(_args.mmrdate) then
+		mmrDisplay = mmrDisplay .. '&nbsp;<small>\'\'(last update: ' .. _args.mmrdate .. '\'\'</small>'
+	end
+
+	return {Cell{name = 'Solo MMR', content = {mmrDisplay}}}
+end
+
+function CustomPlayer:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomPlayer:adjustLPDB(lpdbData)
+	lpdbData.extradata.role = Variables.varDefault('role')
+	lpdbData.extradata.role2 = Variables.varDefault('role2')
+	lpdbData.extradata.lc_id = mw.ustring.lower(mw.title.getCurrentTitle().prefixedText)
```

Comment:
Something tell me this is a copy-paste as well

Reply:
```suggestion
```
yeah it is unused